scripts/test/hyriseBenchmarkTPCH_test.py

```diff
@@ -16,7 +16,7 @@ def main():
   arguments["--use_prepared_statements"] = "true"
   arguments["--queries"] = "'1,13,19'"
   arguments["--time"] = "10"
-  arguments["--runs"] = "100"
+  arguments["--runs"] = "-1"
```

Comment:
Remove, since this is now back to default of benchmark config?

Reply:
The idea is to see what is tested in this test

Reply:
I do not like redundancy, which is why I brought it up. I am OK with this as well.

Reply:
I wouldn't call it redundant. If this was a defaulted c++ parameter, I would also want to explicitly test the parameter, not relying on whatever was set as a default